### PR TITLE
Added netcup DNS API

### PR DESCRIPTION
**Pull request by linux-insideDE**
Please add netcup DNS API to acme.sh

thx
linux-insideDE

### CODE REVIEW

File: dnsapi/dns_netcup.sh [hunk 1]
```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env sh
+
+#Requirments: jq
```
Comment (Neilpang) [inline dnsapi/dns_netcup.sh]: do not use jq

File: dnsapi/dns_netcup.sh [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		if [ "$tmp" != "" ]; then
+			if [ "$tld" = "" ]; then
+				tld=$tmp						
+			else
+				domain=$tmp
+				exit=$i
+				break;
+			fi
+		fi		
+		i=$((i - 1))
```
Comment (Neilpang) [inline dnsapi/dns_netcup.sh]: please use `_math()` function instead.

File: dnsapi/dns_netcup.sh [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	done	
+	inc=""
+	i=1	
+	while [ "$i" -lt "$exit" ];
+	do
+		if [ "$((exit-1))" = "$i" ]; then
```
Comment (Neilpang) [inline dnsapi/dns_netcup.sh]: what is it ?

File: dnsapi/dns_netcup.sh [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+	if [ "$(echo "$tmp" | jq -r .status)" != "success" ]; then
+		_err "$tmp"
+		return 1
+	fi
+}
+getRecords() {	
+	tmp2=$(_post "{\"action\": \"infoDnsRecords\", \"param\": {\"apikey\": \"$NC_Apikey\", \"apisessionid\": \"$sid\", \"customernumber\": \"$NC_CID\", \"domainname\": \"$1\"}}" "$end" "" "POST")
+	xxd=$(echo "$tmp2" | jq -r ".responsedata.dnsrecords" | tr '[' ' ' | tr ']' ' ')
+	xcd=$(echo "$xxd" | sed 's/}\s{/},{/g')
+	echo "[ $xcd ]"
```
Comment (Neilpang) [inline dnsapi/dns_netcup.sh]: what is it for ?

File: dnsapi/dns_netcup.sh (hunk 2 above)
Comment (linux-insideDE) [reply dnsapi/dns_netcup.sh]: done thx

**Comment by c0zm0z** [comment]
Are there any news about the pull request?

**Comment by bohrsty** [comment]
I'm interested too, is there any progress?

**Comment by Neilpang** [comment]
please merge the latest code.

**Comment by linux-insideDE** [comment]
done.

**Comment by Neilpang** [comment]
sorry,  please merge the latest code again.

**Comment by linux-insideDE** [comment]
done again ;)

**Pull request merged**